src/utils.js

```diff
@@ -141,6 +141,31 @@ export const reactHTMLTags = [
 
 
 /**
+ * List of "void element" tags
+ * lifted from React source
+ * https://github.com/facebook/react/blob/6a659606415d4853026df6e6d525274fdc7d35ea/src/renderers/dom/shared/ReactDOMComponent.js#L443
+ */
+export const voidElementTags = {
```

Comment:
I thought about this a bit and we discussed offline.... I think it's fine as-is for now.  The setup you suggest is little bit more "comfortingly structural," but I'm not sure the current complexity of what we need to accomplish necessitates it.  The changes I made have the advantage of providing lucidity into purpose.  We also don't expect that HTML specs will change that frequently.  I'm going to go ahead and keep this as-is but we may run into a compelling reason for a setup like this in the future.